### PR TITLE
ref: 177209747 Adds new entity for payment transcation.

### PR DESCRIPTION
**Pull request by ymhari**

### CODE REVIEW

File: lib/aca_entities/payment_transactions/contracts/transaction_contract.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Validates the presence of params and type of effective date.
+module AcaEntities
+  # Payment Transaction
+  module PaymentTransactions
```
Comment (saikumar9) [inline lib/aca_entities/payment_transactions/contracts/transaction_contract.rb:6]: Can you change the naming to AcaEntities::Contracts::PaymentTransactionContract?

File: lib/aca_entities/payment_transactions/transaction.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module PaymentTransactions
```
Comment (saikumar9) [inline lib/aca_entities/payment_transactions/transaction.rb:4]: Also, here to AcaEntities::PaymentTransaction.

**Review by saikumar9 — CHANGES_REQUESTED** [review]

File: lib/aca_entities/payment_transactions/contracts/transaction_contract.rb (hunk 1 above)
Comment (ymhari) [reply lib/aca_entities/payment_transactions/contracts/transaction_contract.rb]: Removed payment transactions namespace.

File: lib/aca_entities/payment_transactions/transaction.rb (hunk 2 above)
Comment (ymhari) [reply lib/aca_entities/payment_transactions/transaction.rb]: Removed payment transactions namespace.

**Review by raghuramg — APPROVED** [review]
Reviewed code and gem spec updates.

Changes looks good.

**Comment by saikumar9** [comment]
This will be handled in PR: https://github.com/ideacrew/aca_entities/pull/5
Branch: **use_case_v2_177209747**

**Pull request closed without merging**